extensions/amp-image-slider/1.0/component.type.js

```diff
@@ -0,0 +1,23 @@
+/** @externs */
+
+/** @const */
+var BentoImageSliderDef = {};
+
+/**
+ * @typedef {{
+ *   initialPosition: (string|undefined),
+ *   displayHintOnce: (boolean|undefined),
+ *   stepSize: (number),
+ * }}
+ */
+BentoImageSliderDef.Props;
+
+/** @interface */
+BentoImageSliderDef.BentoImageSliderApi = class {
+  /**
+   * Seeks Bar to specified percentage in slider.
+   * 0-49 for right side image, 50 for both image (as half) and 51-100 for left side image.
+   * @param {int} percent Percentage to seek
```

Comment:
```suggestion
   * @param {number} percent Percentage to seek
```

Reply:
0.1 states that the percentage is taken as [0, 1] not [0, 100]. Please update accordingly https://amp.dev/documentation/components/amp-image-slider/#actions